lib/client.js

```diff
@@ -409,11 +395,15 @@ extend(Raven.prototype, {
     return result;
   },
   captureError: function captureError() {
-    utils.consoleAlert('captureError has been deprecated and will be removed in v2.0');
+    utils.consoleAlert('captureError has been deprecated and will be removed in v2.0; use captureException instead');
     return this.captureException.apply(this, arguments);
   },
+  captureQuery: function captureQuery() {
+    utils.consoleAlert('captureQuery has been deprecated and will be removed in v2.0');
```

Comment:
> will be removed

Doesn't this straight-up remove it? Why not keep the original implementation w/ this warning and remove in 2.0?